src/software/simulated_tests/friendly_goal_scored.py

```diff
@@ -0,0 +1,51 @@
+import software.python_bindings as tbots
+from proto.import_all_protos import *
+
+from software.simulated_tests.validation import (
+    Validation,
+    create_validation_geometry,
+    create_validation_types,
+)
+
+
+class FriendlyGoalScored(Validation):
+
+    """Checks if a ball enters the enemy goal."""
+
+    def __init__(self):
+        self.region = tbots.Field.createSSLDivisionBField().enemyGoal()
+
+    def get_validation_status(self, world) -> ValidationStatus:
+        """Checks if the ball enters the provided regions
+
+        :param world: The world msg to validate
+        :returns: FAILING until a ball enters the enemy goal
+                  PASSING when a ball enters
+        """
+
+        if tbots.contains(
+            self.region, tbots.createPoint(world.ball.current_state.global_position)
+        ):
+            return ValidationStatus.PASSING
```

Comment:
this class is pretty similar to BallEntersRegion. You could probably extend that class and just override the regions field to be the enemy goal

Reply:
Deleted this file as it isn't used anywhere. There's another class called FriendTeamScored that seems to do the same thing.